README.md

```diff
@@ -19,8 +19,8 @@ Copyright © 2009- The Spyder Development Team.
 **QtPy** is a small abstraction layer that lets you
 write applications using a single API call to either PyQt or PySide.
 
-It provides support for PyQt5, PyQt4 and PySide using the PyQt5 layout (where
-the QtGui module has been split into QtGui and QtWidgets).
+It provides support for PyQt5, PyQt4, PySide2 and PySide using the Qt5 layout
+(where the QtGui module has been split into QtGui and QtWidgets).
 
 Basically, you write your code as if you were using PyQt5 but import Qt modules
```

Comment:
@ccordoba12 @rlaverde we should change this to 

Basically, you write your code as if you were using PySide2 but import Qt modules from `qtpy` instead of `PySide2` (or PyQt5)

Which is what we actually do, because we use Signal instead of (`PyQt5Signal`) for instance....